packages/documentsearch/src/providers/codemirrorsearchprovider.ts

```diff
@@ -328,7 +390,7 @@ export class CodeMirrorSearchProvider implements ISearchProvider {
   private _findNext(reverse: boolean): Private.ICodeMirrorMatch {
     return this._cm.operation(() => {
       const caseSensitive = this._query.ignoreCase;
-      const cursorToGet = reverse ? 'from' : 'to';
+      const cursorToGet = reverse ? 'from' : 'head';
```

Comment:
From the codemirror docs, this could be 'from' or 'to', or 'head' or 'anchor'. It seems that choosing between 'from' and 'head' is mixing up the possible choices, as those aren't really opposites. What was the reason to change this from 'to' to 'head', but not change 'from' to 'anchor'?

Reply:
Answered in a comment in the latest commit.